### ISSUE TITLE
Participants are looking for a "staking" option on the Governance website, but struggled to find one

### ISSUE THREAD

**Issue by campbellssource** [vegaprotocol/frontend-monorepo#3091]
## Description

This is a usability bug but it _MAY_ have been resolved (in part or in full) by the recent changes to show 'my stake' in the validator table??

Participants set about the task of staking their tokens were often lost for a compelling link to click on.

<img width="1494" alt="Screenshot 2023-03-06 at 11 37 27" src="https://user-images.githubusercontent.com/13255539/223099769-cb57bdf0-250e-4117-a435-886b94191948.png">

The Validators link at the top of the page got the most traction but users were not 100% convinced it was the right place to start.

Some found the link to staking in the side bar.

Once on the validators page some struggled to know what to do next. https://github.com/vegaprotocol/frontend-monorepo/issues/2980

Once staked Some users wanted a page to go to that showed them their stake and how it was distributed (the side bar summary was acceptable but not really what people where after)

----

## Old out of date sketches

<img width="1233" alt="Screenshot 2023-03-06 at 11 45 32" src="https://user-images.githubusercontent.com/13255539/223101444-4282df60-0b9e-4b25-b15c-96b28a342838.png">

in this page the Your stake was an input service the purpose of both a list of validators and a place where your stake could be managed.

The following sketch are more about showing the rewards from staking (and an APY alternative), and selecting a validator would come in another step.

<img width="967" alt="Screenshot 2023-03-06 at 11 47 47" src="https://user-images.githubusercontent.com/13255539/223101965-2b8659eb-010c-4fa7-9d6f-3226260c2887.png">

<img width="784" alt="Screenshot 2023-03-06 at 11 50 07" src="https://user-images.githubusercontent.com/13255539/223102327-c65de1c3-9d8a-4f2b-b5ce-fdafe67c8157.png">

---

##TL:DR;

Consider re-naming top nav item from Validators to Staking

Consider repurposing the staking page to make it more useful to manage your own stake
